sites/public/pages/listings.tsx

```diff
@@ -52,6 +74,106 @@ export default function ListingsPage(props: ListingsProps) {
       </Head>
       <MetaTags title={t("nav.siteTitle")} image={metaImage} description={metaDescription} />
       <PageHeader title={t("pageTitle.rent")} />
+      <Modal
+        open={filterModalVisible}
+        title={"Filter Results"}
+        ariaDescription={"testing aria"}
+        actions={[
+          <Button
+            onClick={() => setFilterModalVisible(false)}
+            styleType={AppearanceStyleType.primary}
+          >
+            Apply
+          </Button>,
+          <Button
+            onClick={() => setFilterModalVisible(false)}
+            styleType={AppearanceStyleType.secondary}
+            border={AppearanceBorderType.borderless}
+          >
+            Close
+          </Button>,
+        ]}
+        hideCloseIcon
+      >
+        <Form onSubmit={handleSubmit(onSubmit)}>
+          <div className="form-card__group">
+            <p className="field-note mb-4">
+              {t("Use these options to refine your list of properties.")}
+            </p>
+            <Select
+              id="filter.unitOptions"
+              name="filter.unitOptions"
+              //placeholder={t("Placeholder text")}
+              label={t("unitOptions.label")}
+              validation={{ required: true }}
+              //defaultValue={t("eligibility.income.ranges")}
+              register={register}
+              controlClassName="control"
+              options={preferredUnitOptions}
+              keyPrefix="unitOptions.unitOptionsTypes"
+            />
+            <Select
+              id="filter.accessibilityOptions"
+              name="filter.accessibilityOptions"
+              label={t("accessibilityOptions.label")}
+              validation={{ required: true }}
+              register={register}
+              controlClassName="control"
+              options={accessibilityOptions}
+              keyPrefix="accessibilityOptions.accessibilityOptionsTypes"
+            />
+            <Select
+              id="filter.communityOptions"
+              name="filter.communityOptions"
+              //placeholder={t("Placeholder text")}
+              label={t("communityOptions.label")}
+              validation={{ required: true }}
+              //defaultValue={t("eligibility.income.ranges")}
+              register={register}
+              controlClassName="control"
+              options={communityOptions}
+              keyPrefix="communityOptions.communityOptionsTypes"
+            />
+          </div>
+        </Form>
+      </Modal>
+      <Drawer
```

Comment:
Yes until we agree which filter view we want (model vs drawer), I was just going to leave both.

Reply:
@pfeeny5 I thought you said we were going with the modal because Drawers are typically bad for UX reasons? I'm comfortable removing the drawer code, that also means I won't have to implement filtering for both cases.